draft-ietf-scitt-architecture.md

```diff
@@ -421,6 +421,88 @@ For interoperability, Registration Policy decisions should be based on interpret
 
 Each implementation of a Transparency Service MAY support additional metadata, specific to its implementation through additional ["Reserved for Private Use"](https://www.iana.org/assignments/cwt/cwt.xhtml#claims-registry) keys within the `CWT_Claims` header.
 
+### Signing Statements with X.509
+
+Signing a Statement with an X.509 certificate includes the following steps:
+
+1. An Issuer makes a Statement about an Artifact
+1. The Issuer will sign the Statement, capturing relevant properties in the protected header (7) and envelope (8)
+1. A Statement is a payload about the Artifact, including a hash of the Artifact, an SBOM or other type of Attestation about the Artifact
+1. Statements are signed and placed in the `payload` of CoseSign1 envelope.
+   The statement content may be stored outside of SCITT for privacy or size constraints, where a hash of the file is used to represent the Statement content.
+1. The Statement hash or Statement content is signed by the Issuer
+1. A CWT_Claims object is constructed, setting the `iss` to the Issuer and the `sub` as an identifier for the Artifact
+1. A Protected Header is constructed, capturing the Algorithm and Key ID, used to sign the Statement
+1. A COSE_Sign1 envelope is constructed aggregating the Protected Header, the CWT_Claims, Signature from the signing process, and the matching payload that was signed (hash or the content)
+1. The Signed Statement is submitted to the Transparency Service where the Registration Policy evaluates the Signed Statement
+1. Upon successful registration, a Receipt is returned as a property in the Unprotected Header of the Signed Statement, which is now a Transparent Statement
```

Comment:
This is not what we believe happens.  We simply can return the receipt, why roundtrip the signed statement and serialize it again with the receipt?

Reply:
Good clarifying point on not round-tripping the Signed Statement and a potentially large payload. See #176 for an update to return receipts.